test/fixtures/output/network/delete-network-not-exist.golden

```diff
@@ -0,0 +1 @@
+Error: The network n-invalid was not found.: 404 Not Found
```

Comment:
Can you update your API responses so they match Go error style (start with lowercase, no period at the end)?

If not, we plan to write a helper function in the CLI to clean up these error strings (you're not the only team who does this)

Reply:
No worries if not, I think a lot of teams prefer uppercase with trailing period so it looks nice on the UI side.

Reply:
we haven't been following strict guidelines in formatting the error messages in API responses.
I'm in favor of standardizing it but it could be a non-trivial effort to fix existing API responses. 
Let's take an AI here and check with the team.